src/Symfony/Component/Form/ChoiceList/Factory/PropertyAccessDecorator.php

```diff
@@ -172,6 +172,11 @@ public function createView(ChoiceListInterface $list, $preferredChoices = null,
 
         if (is_string($label) && !is_callable($label)) {
             $label = new PropertyPath($label);
+        } elseif (is_string($label) && is_callable($label)) {
+            // Prevent a fatal error since a callable string may not handle the right arguments
```

Comment:
@stof Actually `choice_label`, `choice_name`, `choice_attr`, `group_by` and `preferred_choices` as callable get passed 3 arguments ($choice, $key, $value) and we got a fatal error with callable such as `'end'` but I found out that was not valid because `'MyStatic::getMeChoiceName'` would then fail if it needs 3 arguments, so this is a "won't fix" and I closed this PR.